### PR TITLE
Project3 tyler

### PR DESCRIPTION
**Pull request by tylerscottarndt**

### CODE REVIEW

File: src/main/java/com/webservices/models/Customer.java [hunk 1]
```diff
@@ -19,6 +20,9 @@
     @Column(name = "NAME")
     private String name;
 
+    @Column(name = "BANK")
```
Comment (RobusK) [inline src/main/java/com/webservices/models/Customer.java]: better name?

File: src/main/java/com/webservices/models/Customer.java [hunk 2]
```diff
@@ -81,6 +89,7 @@ public boolean equals(Object o) {
         Customer customer = (Customer) o;
         return Objects.equals(id, customer.id) &&
                 Objects.equals(name, customer.name) &&
+                Objects.equals(bank, customer.bank) &&
```
Comment (RobusK) [inline src/main/java/com/webservices/models/Customer.java]: hashcode?

**Comment by tylerscottarndt** [comment]
Ah, I'm still having problems with this bean issue. I wrote as many tests as I could, hopefully correctly, as it's my first time trying Mockito. I'm available to get together most of the day tomorrow to go over solutions.

File: src/main/java/com/webservices/models/Product.java [hunk 3]
```diff
@@ -107,6 +114,7 @@ public boolean equals(Object o) {
                 Objects.equals(name, product.name) &&
                 Objects.equals(description, product.description) &&
                 Objects.equals(price, product.price) &&
+                Objects.equals(quantity, product.quantity) &&
```
Comment (RobusK) [inline src/main/java/com/webservices/models/Product.java]: hashcode?

File: src/main/java/com/webservices/models/Supplier.java [hunk 4]
```diff
@@ -19,6 +20,9 @@
     @Column(name = "NAME")
     private String name;
 
+    @Column(name = "BANK")
```
Comment (RobusK) [inline src/main/java/com/webservices/models/Supplier.java]: better name?

File: src/main/java/com/webservices/models/Supplier.java [hunk 5]
```diff
@@ -48,6 +56,27 @@ public void setProducts(List<Product> products) {
...
+    public void fulfillParcel(Parcel parcel){
+        List<Product> products = parcel.getProducts();
+        List<Product> fulfilledProducts = new ArrayList<Product>();
+        for(Product product : products){
+            int quantity = product.getQuantity();
+            if(quantity > 0){
+                product.setQuantity(quantity - 1);
+                fulfilledProducts.add(product);
+            } else {
+                //error handling
```
Comment (RobusK) [inline src/main/java/com/webservices/models/Supplier.java]: throw an exception?

**Pull request merged**